env_setup/env_cleanup.yml

```diff
@@ -22,45 +23,18 @@
         # Clean up router VM, vSwitch and portgroup
         - include_tasks: ../common/network_testbed_cleanup.yml
       when:
+        - not cleanup_vm | bool
```

Comment:
Even `cleanup_vm` is set to True, we also need to cleanup deployed router VM and portgroup on existing testbed.

Reply:
Yes, you are right. I just removed this line. Thanks~